qiskit_ibm/api/rest/runtime.py

```diff
@@ -140,7 +140,7 @@ def program_run(
             'group': group,
             'project': project,
             'backend': backend_name,
-            'params': [params],
+            'params': params,
             'runtime': image
         }
         data = json.dumps(payload)
```

Comment:
This dump will fail if `params` has non JSON serializable data, which most of our programs have. That's why the old code dumps with `RuntimeDecoder` (which converts at least all the params used by our programs)